config/peerpods/podvm/bootc/Containerfile.rhel

```diff
@@ -0,0 +1,82 @@
+# build pause
+FROM registry.k8s.io/pause:3.9 as pause-bin
+
+FROM registry.redhat.io/ubi9/ubi-minimal:9.5 as pause
+RUN curl -L https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64 -o /usr/bin/umoci && chmod +x /usr/bin/umoci
+RUN umoci init --layout pause && umoci new --image pause:k8s
+RUN umoci config --image pause:k8s --config.entrypoint=/pause --author="OSC"
+RUN umoci unpack --rootless --image pause:k8s /pause
+COPY --from=pause-bin /pause /pause/rootfs/pause
+
+# Get payload
+FROM brew.registry.redhat.io/rh-osbs/openshift-sandboxed-containers-podvm-payload:osc-1.9-rhel-9-containers-candidate-96969-20250224095457 as payload
+
+# Build bootc rhel podvm
+FROM registry.redhat.io/rhel9/rhel-bootc:9.5-1738698007 as podvm-bootc
+
+ARG ORG_ID
+ARG ACTIVATION_KEY
+
+# register
+RUN if [[ -n "${ACTIVATION_KEY}" && -n "${ORG_ID}" ]]; then \
+    #rm -f /etc/rhsm-host && rm -f /etc/pki/entitlement-host; \
+    subscription-manager register --org=${ORG_ID} --activationkey=${ACTIVATION_KEY}; \
+    fi
+
+COPY etc /etc
+COPY usr /usr
+
+# afterburn is required for Azure
+#RUN dnf config-manager --add-repo=https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/ && dnf install -y --nogpgcheck afterburn && dnf clean all && dnf config-manager --set-disabled "*centos*"
+RUN dnf install -y afterburn && dnf clean all
```

Comment:
Installing afterburn is the only thing requires a subscription and AFAIK, required only by Azure.
Should we drop it? this will allow building bootc image in-cluster using the buildconfig definition without any special configuration.
@beraldoleal do you know if afterburn (or any other pkg) is required for gcp?

Reply:
@snir911 I haven’t used it directly during development, so my initial assumption is that it’s not needed and can be dropped. If it was used, it must have been under the hood, and I didn’t notice.

That said, we can revisit this if we run into any issues with GCP later.

Reply:
Afaik afterburn is used specifically for azure in mkosi build today to indicate that the VM is ready.
https://github.com/confidential-containers/cloud-api-adaptor/commit/a70e147768d8026f4fe67ea45008355c165408f5

Reply:
@snir911 what if have an argument (`CLOUD_PROVIDER`) that if "azure" then subscribe && install afterburn?